apps/participants/views.py

```diff
@@ -0,0 +1,123 @@
+    model = Participant
+    form_class = ParticipantForm
+    template_name = "generic/object_form.html"
+    extra_context = {
+        "title": "Update Participant",
+        "button_text": "Update",
+        "active_tab": "participants",
+    }
+
+    def get_queryset(self):
+        return Participant.objects.filter(team=self.request.team)
+
+    def get_success_url(self):
+        return reverse("participants:participant_home", args=[self.request.team.slug])
+
+
+class DeleteParticipant(LoginAndTeamRequiredMixin, View, PermissionRequiredMixin):
+    permission_required = "experiments.delete_participant"
+
+    def delete(self, request, team_slug: str, pk: int):
+        messages.error(request, "Cannot delete a Participant")
+        return HttpResponse()
+
+
+class ParticipantTableView(SingleTableView):
+    model = Participant
+    paginate_by = 25
+    table_class = ParticpantTable
+    template_name = "table/single_table.html"
+
+    def get_queryset(self):
+        return Participant.objects.filter(team=self.request.team)
+
+
+class SingleParticipantHome(LoginAndTeamRequiredMixin, TemplateView, PermissionRequiredMixin):
+    permission_required = "experiments.view_participant"
+    template_name = "participants/single_participant_home.html"
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        participant = Participant.objects.prefetch_related("experimentsession_set").get(
+            id=self.kwargs["participant_id"]
+        )
+        context["active_tab"] = "participants"
+        context["participant"] = participant
+        experiment_data = {}
+        for experiment in participant.get_experiments_for_display():
+            sessions = participant.experimentsession_set.filter(experiment=experiment).all()
+            experiment_data[experiment] = {
+                "sessions": sessions,
+                "participant_data": json.dumps(sessions.first().participant_data_from_experiment, indent=4),
+            }
+        context["experiment_data"] = experiment_data
+        return context
+
+
+class EditParticipantData(LoginAndTeamRequiredMixin, TemplateView, PermissionRequiredMixin):
+    def post(self, request, team_slug, participant_id, experiment_id):
+        experiment = Experiment.objects.get(team__slug=team_slug, id=experiment_id)
+        participant = Participant.objects.get(id=participant_id)
+        new_data = json.loads(request.POST["data"])
+        ParticipantData.objects.update_or_create(
+            participant=participant,
+            content_type__model="experiment",
+            object_id=experiment_id,
+            team=request.team,
+            defaults={"team": experiment.team, "data": new_data, "content_object": experiment},
+        )
+        return HttpResponseRedirect(
+            reverse("participants:single-participant-home", args=[self.request.team.slug, participant_id])
+        )
```

Comment:
nit: use django shortcut:
```suggestion
        return redirect("participants:single-participant-home", self.request.team.slug, participant_id)
```